app/controllers/api/v1/applied_coupons_controller.rb

```diff
@@ -35,6 +56,12 @@ def create_params
           :percentage_rate,
         )
       end
+
+      def valid_status?(status)
+        return false unless status
+
+        AppliedCoupon.statuses.keys.include?(status)
```

Comment:
Performance/InefficientHashSearch: Use `#key?` instead of `#keys.include?`. (https://github.com/JuanitoFatas/fast-ruby#hashkey-instead-of-hashkeysinclude-code)